js/ui/components/clubhouse.js

```diff
@@ -198,6 +204,74 @@ var ClubhouseNotificationBanner = new Lang.Class({
         }
     },
 
+    _rearrangeElements: function() {
+        let contentBox = this._bodyStack.get_parent();
+        contentBox.add_style_class_name('clubhouse-content-box');
+        let hbox = contentBox.get_parent();
+        let vbox = hbox.get_parent();
+
+        let wrapBin = new St.Bin({ x_fill: true,
+                                   y_fill: true,
+                                   x_align: St.Align.END,
+                                   y_align: St.Align.END,
+                                   x_expand: true,
+                                   y_expand: true });
+        hbox.add_child(wrapBin);
+
+        // A Clutter.BinLayout is used to rearrange the notification
+        // elements in layers:
+        let wrapWidget = new St.Widget({ layout_manager: new Clutter.BinLayout(),
+                                         x_align: Clutter.ActorAlign.FILL,
+                                         y_align: Clutter.ActorAlign.END });
+        wrapBin.set_child(wrapWidget);
+
+        let iconBinParams = { x_fill: false,
+                              y_fill: false,
+                              x_expand: true,
+                              y_expand: true,
+                              x_align: Clutter.ActorAlign.END,
+                              y_align: Clutter.ActorAlign.END };
+        Object.assign(this._iconBin, iconBinParams);
+
+        let contentBoxParams = { x_expand: true,
+                                 y_expand: false,
+                                 x_align: Clutter.ActorAlign.FILL,
+                                 y_align: Clutter.ActorAlign.END };
+        Object.assign(contentBox, contentBoxParams);
+
+        let actionBinParams = { x_expand: true,
+                                y_expand: true,
+                                x_align: Clutter.ActorAlign.START,
+                                y_align: Clutter.ActorAlign.END };
+        Object.assign(this._actionBin, actionBinParams);
+
+        let expandedLabelActorParams = { y_expand: true,
+                                         y_align: Clutter.ActorAlign.CENTER };
+        Object.assign(this._expandedLabel.actor, expandedLabelActorParams);
+        this._expandedLabel.actor.clutter_text.line_wrap_mode = Pango.WrapMode.WORD_CHAR;
+
+        hbox.remove_child(contentBox);
+        wrapWidget.add_child(contentBox);
+
+        hbox.remove_child(this._iconBin);
+        wrapWidget.add_child(this._iconBin);
+
+        vbox.remove_child(this._actionBin);
+        wrapWidget.add_child(this._actionBin);
+    },
+
+    // Override the callback that changes the button opacity on hover:
+    _sync: function() {
+        this._closeButton.opacity = 255;
+    },
+
+    _resetTimeout: function() {
```

Comment:
I think this function got added here by mistake. I will remove it before merging it.

Reply:
Yeah @cosimoc that was my fault whlie rebasing. thanks for noticing.